buildpacks/php/tests/integration/smoke.rs

```diff
@@ -98,5 +98,18 @@ fn smoke_test_php_polyfills() {
         .target_triple(target_triple(builder()))
         .to_owned();
 
-    TestRunner::default().build(&build_config, |_context| {});
+    TestRunner::default().build(&build_config, |context| {
+        assert_contains!(
+            context.pack_stderr,
+            r#"composer require "heroku-sys/ext-mcrypt.native:*""#
+        );
+        assert_contains!(
+            context.pack_stderr,
+            r#"composer require "heroku-sys/ext-ctype.native:*""#
+        );
+        assert_contains!(
+            context.pack_stderr,
+            r#"composer require "heroku-sys/ext-mbstring.native:*""#
+        );
```

Comment:
`s/pack_stderr/pack_stdout/` due to:
https://github.com/heroku/cnb-builder-images/pull/784#issuecomment-3188677609

```suggestion
        assert_contains!(
            context.pack_stdout,
            r#"composer require "heroku-sys/ext-mcrypt.native:*""#
        );
        assert_contains!(
            context.pack_stdout,
            r#"composer require "heroku-sys/ext-ctype.native:*""#
        );
        assert_contains!(
            context.pack_stdout,
            r#"composer require "heroku-sys/ext-mbstring.native:*""#
        );
```